MenuAl/MenuAl/SceneDelegate.swift

```diff
@@ -16,7 +16,11 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         // Use this method to optionally configure and attach the UIWindow `window` to the provided UIWindowScene `scene`.
         // If using a storyboard, the `window` property will automatically be initialized and attached to the scene.
         // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
-        guard let _ = (scene as? UIWindowScene) else { return }
+        guard let windowScene = (scene as? UIWindowScene) else { return }
```

Comment:
Bu satiri aciklar misin?

Reply:
ilk satırda sadece pencerenin UIWindowScene olup olmadığını kontrol ediyor ve burda sonuç değerini kullanmıyor.
İkinci satırda ise pencerenin UIWindowScene olduğundan emin olduktan sonra, bu nesneyi daha fazla işlem yapmak için kullanır.
Kısaca ( guard let _ ) : satırı dönüştürülen değeri kullanmıyor, sadece kontrol için kullanıyor.
( guard let windowScene ) : Dönüştürülen değeri bir değişkene atıyor ve kullanıma hazır hale getiriyor